### PR TITLE
IBX-8534: Change deprecated path to inputUri 

### PR DESCRIPTION
**Pull request by ViniTou**
| :ticket: Issue | IBX-8534 |
|----------------|-----------|

#### Related PRs: 
- https://github.com/ibexa/core/pull/456

### CODE REVIEW

**Review by Steveb-p — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

File: .github/workflows/ci.yaml
Comment (alongosz) [inline .github/workflows/ci.yaml:37]: Please include in the PR description clear and visible message that the tmp commit needs to be dropped before merging.

**Review by alongosz — COMMENTED** [review]
Looks ok, I just need a little bit more ensurance to make sure we won't merge this with tmp commits, just to be safe:

**Review by alongosz — APPROVED** [review]

**Pull request merged**
